zap/network_generated.go

```diff
@@ -0,0 +1,196 @@
+// Zed Attack Proxy (ZAP) and its related class files.
```

Comment:
Better remove this one, most of the endpoints will not work with 2.11.

Reply:
Sorry I should have caught that, excellent point.

Reply:
This exists on the python API as well.